sci-libs/lib2bit/lib2bit-0.2.1.ebuild

```diff
@@ -17,15 +17,20 @@ IUSE="static"
 DEPEND=""
 RDEPEND="${DEPEND}"
 
+PATCHES=(
+	"${FILESDIR}"/${P}_respect_LDFLAGS.patch
+	"${FILESDIR}"/${P}_test_support_python3.patch
+)
+
 src_prepare(){
 	default
-	epatch "${FILESDIR}"/${P}_respect_DESTDIR.patch
-	sed -e 's#/usr/local#/usr#' -i Makefile || die
```

Comment:
Is it taken care of elsewhere?

Reply:
We no longer use the make install target, but rather install the libraries and headers by ourselves.